samples/bluetooth/peripheral_hids_mouse/README.rst

```diff
@@ -1,24 +1,134 @@
+-----------------------------------------
+
+To test with a Microsoft Windows computer that has a Bluetooth radio, complete the following steps:
+
+1. Power on your development board.
+#. On your Windows computer, search for Bluetooth devices and connect to the device named "NCS HIDS mouse".
+#. Push Button 1 on the board.
+   Observe that the mouse pointer on the computer moves to the left.
+#. Push Button 2.
+   Observe that the mouse pointer on the computer moves upward.
+#. Push Button 3.
+   Observe that the mouse pointer on the computer moves to the right.
+#. Push Button 4.
+   Observe that the mouse pointer on the computer moves downward.
+#. Disconnect the computer from the device by removing the device from the computer's devices list.
+
+
+Testing with nRF Connect for Desktop
+------------------------------------
+
+To test with `nRF Connect for Desktop`_, complete the following steps:
+
+1. Power on your development board.
+#. Connect to the device from nRF Connect (the device is advertising as "NCS HIDS mouse"), then bond to the device.
+   To bond, click the settings button for the device in nRF Connect, select **Pair**, check **Perform Bonding**, and click **Pair**.
+   Wait until the bond is established before you continue.
+   Observe that the services of the connected device are shown.
+#. Click the **Play** button for all HID Report characteristics.
+#. Push Button 1 on the board.
+   Observe that a notification is received on one of the HID Report characteristics, containing the value ``FB0F00``.
+
+   Mouse motion reports contain data with an X-translation and a Y-translation.
+   These are transmitted as 12-bit signed integers.
+   The format used for mouse reports is the following byte array, where LSB/MSB is the least/most significant bit: ``[8 LSB (X), 4 LSB (Y) | 4 MSB(X), 8 MSB(Y)]``.
+
+   Therefore, ``FB0F00`` denotes an X-translation of FFB = -5 (two's complement format), which means a movement of 5 pixels to the left, and a Y-translation of 000 = 0.
+#. Push Button 2.
+   Observe that the value ``00B0FF`` is received on the same HID Report characteristic.
+#. Push Button 3.
+   Observe that the value ``050000`` is received on the same HID Report characteristic.
+#. Push Button 4.
+   Observe that the value ``005000`` is received on the same HID Report characteristic.
+#. Disconnect the device in nRF Connect.
+   Observe that no new notifications are received and the device is advertising.
+#. As bond information is preserved by nRF Connect, you can immediately reconnect to the device by clicking the Connect button again.
+
+
+Dependencies
+************
+
+This sample uses the following |NCS| libraries:
+
+* :ref:`hids_readme`
+
+In addition, it uses the following Zephyr libraries:
+
+* ``include/zephyr/types.h``
+* ``include/misc/printk.h``
+* ``include/misc/byteorder.h``
+* :ref:`GPIO Interface <zephyr:io_interfaces>`
+* :ref:`zephyr:settings`
+* :ref:`zephyr:bluetooth_api`:
```

Comment:
What about `assert.h,` `errno.h` and `soc.h` ?

Reply:
I cannot find any of those - so I assumed they are just standards. ;)
Is it helpful for the user to list them? And if so, where are the headers?

Reply:
They are just standards (present in most of the NCS samples), but zephyr/types.h, printk.h and byteorder.h are also quite standard

Reply:
Guess it doesn't hurt to list them. Are they in the Zephyr repo (if so, where)?

Reply:
Thx - I added errno and assert.